packages/nc-gui/components/virtual-cell/QrCode.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import { useQRCode } from '@vueuse/integrations/useQRCode'
+
+const cellValue = inject(CellValueInj)
+
+const qrValue = computed(() => String(cellValue?.value))
+const qrCode = useQRCode(qrValue, {
+  width: 150,
+})
+const qrCodeLarge = useQRCode(qrValue, {
+  width: 600,
+})
+
+const modalVisible = ref(false)
+const showQrModal = (ev: MouseEvent) => {
+  ev.stopPropagation()
+  modalVisible.value = true
+}
+
+const handleModalOkClick = () => (modalVisible.value = false)
+</script>
+
+<template>
+  <a-modal v-model:visible="modalVisible" :title="qrValue" footer @ok="handleModalOkClick" :bodyStyle="{ padding: '0px' }">
+    <img v-if="qrValue" :src="qrCodeLarge" alt="QR Code" class="qr-code" />
+  </a-modal>
```

Comment:
title may overlap with the cancel button

<img width="440" alt="image" src="https://user-images.githubusercontent.com/35857179/202608468-4825ccda-a6b7-4a96-8700-477e2067327f.png">